functions/activityStepManager/src/app/lib/eventMapper.js

```diff
@@ -155,7 +157,13 @@ async function processInvoice(event, recIdx) {
         ]);
         if (timelineElements && timelineElements.length > 0) {
           for (const timelineElem of timelineElements) {
-            invoicedElements.push(processInvoicedElement(timelineElem));
+            invoicedElements.push(
+              processInvoicedElement(
+                timelineElem,
+                // we don't want the timestamp of the timelineElem, but the one of the timelineObj (the one the perfectionated the notification and started the invoice process)
+                invoicedElement.invoincingTimestamp
```

Comment:
@FeliceMente questo è un typo oppure si chiama veramente "invoi**n**cingTimestamp"?

Reply:
il campo è effettivamente quello in tabella. C'è anche un altro campo col typo ed è sort key quindi per il momento va mantenuto così. Ho aggiornato solo il nome del campo passato e messo commenti in corrispondenza dei campi con typos.